Remove Avalanche Wallet references (#1595)
ava-labs/avalanche-docs: docs/nodes/run/third-party/aws-marketplace-one-click.md

}
```

> ## Add Node as Validator on Fuji via Core web

Reviewer: ```suggestion
## Add Node as Validator on Fuji via Core Web
```

For adding the new node as a Validator on the Fuji testnet's Primary Network you can

> use the [Core web](https://core.app/) 

Reviewer: ```suggestion
use the [Core Web](https://core.app/) 
```

[connected](https://support.avax.network/en/articles/6639869-core-web-how-do-i-connect-to-core-web) to
[Core extension](https://join.core.app/extension).
If you don't have a Core extension already, check out this
[guide](https://support.avax.network/en/articles/6100129-core-extension-how-do-i-create-a-new-wallet).
If you'd like to import an existing wallet to Core extension, follow 
[these steps](https://support.avax.network/en/articles/6078933-core-extension-how-do-i-access-my-existing-account).

![Core web](/img/one-click-validator-node/core-web.png)

> Core web is a free, all-in-one command center giving users a more intuitive and comprehensive way to view,

Reviewer: ```suggestion
Core Web is a free, all-in-one command center that gives users a more intuitive and comprehensive way to view,
```

Author: fixed 798fcbef0fd3855c86d950a2af92d9b3ba43f1a4

> and use across the Avalanche network, Avalanche Subnets, and Ethereum. 

Reviewer: ```suggestion
and use dApps across the Avalanche network, its various Subnets, and Ethereum. 
```

Author: fixed 798fcbef0fd3855c86d950a2af92d9b3ba43f1a4

> Core web is optimized for use with Core browser extension and Core mobile (iOS, Android), free, 

Reviewer: ```suggestion
Core Web is optimized for use with the Core browser extension and Core mobile (available on both iOS & Android).
```

> personal wallet (self-custodial). Together, they are key components of the Core product 

Reviewer: ```suggestion
Together, they are key components of the Core product 
```

Author: fixed https://support.avax.network/en/?q=core+mobile

suite that brings dApps, NFTs, Avalanche Bridge, Subnets, L2s, and more, directly to users.

### Switching to Testnet Mode

> By default, Core web and Core extension are connected to Mainnet. 

Reviewer: ```suggestion
By default, Core Web and Core extension are connected to Mainnet. 
```

#### On Core extension

From the hamburger menu on the top-left corner, choose Advanced, and then toggle the Testnet Mode on.

<img src="/img/one-click-validator-node/testnet-core-ext.gif" width="300px"/>

You can follow the same steps for switching back to Mainnet.

#### On Core web

Click on the Settings button top-right corner of the page, then toggle Testnet Mode on.

<img src="/img/one-click-validator-node/testnet-core-web.gif"/>

You can follow the same steps for switching back to Mainnet.

### Adding the Validator

:::info
* Node ID: A unique ID derived from each individual node’s staker certificate. 
Use the `NodeID` which was returned in the `info.getNodeID` response.
In this example it's `NodeID-Q8Gfaaio9FAqCmZVEXDq9bFvNPvDi7rt5`
* Staking End Date: Your AVAX tokens will be locked until this date.
* Stake Amount: The amount of AVAX to lock for staking. On Mainnet,
the minimum required amount is 2,000 AVAX. On Testnet the minimum required
amount is 1 AVAAX.
* Delegation Fee: You will claim this % of the rewards from the delegators on
your node.
* Reward Address: A reward address is the destination address of the accumulated
staking rewards.
:::

> To add a node as a Validator, first select the Stake tab on Core web, 

Reviewer: ```suggestion
To add a node as a Validator, first select the Stake tab on Core Web, 
```
